### PR TITLE
Rename fig to compose

### PR DESCRIPTION
**Pull request by bfirsh**
It's now at https://github.com/docker/compose

### CODE REVIEW

**Comment by jfrazelle** [comment]
wowowowowow \o/ :fireworks:

**Comment by tiborvass** [comment]
I suggest we test this by doing a docs release to beta-docs

**Comment by jfrazelle** [comment]
LGTM

**Comment by SvenDowideit** [comment]
LGTM @fredlf @jamtur01 @spf13 @moxiegirl

File: README.md [hunk 1]
```diff
@@ -239,6 +239,6 @@ server for Docker (hosting/delivering of repositories and images)
 for a container-centric world 
 * [Docker Swarm](https://github.com/docker/swarm): A Docker-native clustering 
 system 
-* [Docker Compose, aka Fig](https://github.com/docker/fig):
```
Comment (jamtur01) [inline README.md]: Previously fig...

Comment (jamtur01) [inline README.md]: Formerly fig...

**Comment by fredlf** [comment]
LGTM, with James' correction. (Sorry, but it ain't Fig no mo!)

**Comment by bfirsh** [comment]
@jfrazelle @fredlf fixed!

**Pull request merged**